frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/storage-and-nodes-step.tsx

```diff
@@ -74,20 +187,50 @@ export const StorageAndNodes: React.FC<StorageAndNodesProps> = ({ state, dispatc
     dispatch({ type: 'setEnableMinimal', value: isMinimal });
   }, [cpu, dispatch, memory, nodesCount]);
 
-  const handleStorageClass = (sc: StorageClassResourceKind) => {
+  const handleStorageClass = (sc: StorageClassResourceKind) =>
     dispatch({ type: 'setStorageClass', value: sc });
-  };
 
   const setNodes = (filteredData: NodeKind[]) =>
     dispatch({ type: 'setNodes', value: filteredData });
 
+  const isArbiterSC = (sc: StorageClassResourceKind): boolean => {
+    const pvs: K8sResourceKind[] = getSCAvailablePVs(pvData, getName(sc));
+    scNodeNames = getAssociatedNodes(pvs);
```

Comment:
This is already calculated above . Why redo ?
```suggestion
```

Reply:
Then, I think , we can consider to move it out and modify the function definition:
```
    const isArbiterSC = (sc: StorageClassResourceKind, nodesData: NodeKind[], scNodeNames: string[]): boolean
```
Since we are already passing `nodesData` in the `StretchClusterFormGroup`,  so it will separate the concerns from parent component and help in following.

```

type stretchClusterFormGroupProps = {
  state: State;
  dispatch: React.Dispatch<Action>;
  isArbiterSC: (Object) => boolean;
  nodesData: NodeKind[];
};
```
And instead of passing `isArbiterSc` , call it directly and pass `scNodeNames` in props:


```

type stretchClusterFormGroupProps = {
  state: State;
  dispatch: React.Dispatch<Action>;
>  scNodeNames: string;
  nodesData: NodeKind[];
};
```